ospf6d/ospf6_neighbor.c

```diff
@@ -981,6 +995,14 @@ static void ospf6_neighbor_show_detail(struct vty *vty,
 			vty_out(vty, "      %s\n", lsa->name);
 
 		bfd_sess_show(vty, NULL, on->bfd_session);
+
+		if (on->auth_present == true) {
+			vty_out(vty, "    Authentication is enabled\n");
```

Comment:
This text is wrong, `on->auth_present` indicates whether an authentication trailer was **received**, not whether it is **enabled/required/checked** for this neighbor.

Reply:
Yes. It means that Authentication is enabled in neighbor. This display it to show that auth trailer is enabled in ospf6 neighbor.

Reply:
`on->auth_present == true` is set when authentication was **received**.  received != enabled.